airflow/models/taskinstance.py

```diff
@@ -65,6 +65,7 @@
 from sqlalchemy.orm.session import Session
 from sqlalchemy.sql.elements import BooleanClauseList
 from sqlalchemy.sql.expression import ColumnOperators, case
+from typing_extensions import Literal, TypeGuard
```

Comment:
`TypeGuard` also presented in our `airflow.typing_compat` so we might want to use two imports here:

```python
from typing_extensions import Literal

from airflow.typing_compat import TypeGuard
```

Reply:
Yes.